templates/main.html

```diff
@@ -10,6 +10,15 @@
 {% endblock %}
 
 {% block content %}
+	{% if 'legacy_welcome' in request.cookies %}
+		<div class="alert alert-warning alert-dismissable" role="alert">
+			<strong>Welcome to the new HTTP Archive!</strong>
```

Comment:
Link "new HTTP Archive" to the post as well?

Reply:
Done

![image](https://user-images.githubusercontent.com/1120896/37983288-099d2322-31c1-11e8-86cb-509258bb8378.png)